app/js/model/filter.js

```diff
@@ -0,0 +1,33 @@
+define([
+  'jquery',
+  'backbone'
+], function ($, Backbone) {
+  return Backbone.Model.extend({
+    defaults: {
+      none: true,
+      done: false,
+      notDone: false
+    },
+
+    setFilterNone: function () {
+      this.set({none: true});
```

Comment:
Better way to do it, but there is a lot of code repetition.

Since we know the filter can only have one value, I would have used a `enum` like structure with a property on the `TodoCollection` class.